src/units/cron-update.service.in

```diff
@@ -4,5 +4,4 @@ Documentation=man:systemd.cron(7)
 
 [Service]
 Type=oneshot
```

Comment:
please remove cron-update.service from MR

The 'systemctl daemon-reload; sleep 1; systemctl restart cron.target' _does_ need to be encapsulated
into a single "sh -c" because systemd won't continue to process ExecStart= in the middle of a daemon-reload.